.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -1,31 +1,55 @@
 ---
 name: "\U0001F41B Bug report"
 about: Create a report to help us fix things
+title: ''
+labels: ['bug', 'needs-triage']
+assignees: ''
+projects: ['nightscout/2']
 
 ---
+## Describe the bug
+*A clear and concise description of what the bug is. Describe what you see versus what you expect to see.*
 
-**Describe the bug**
-A clear and concise description of what the bug is.
+## Attach a Log
+*Tap the Trio settings icon at the bottom of the screen, then tap 'Share logs' on the bottom of the list and attach it to this ticket.*
 
-**To Reproduce**
-Steps to reproduce the behavior:
-1. Go to '...'
-2. Click on '....'
-3. Scroll down to '....'
-4. See error
+## To Reproduce
+*Steps to reproduce the behavior:*
+1. *Go to '...'*
```

Comment:
I don't think listing in bullet form is appropriate for a numbered list. The outcome is an ordered-unordered list which is kind of funky.

Reply:
you mean the '1.' needs to be '1' ?

Reply:
I think you are referring to the star, but that is just text styling...

Or the outdated picture of it. Will refresh that.